core/src/main/resources/hudson/ProxyConfiguration/config.groovy

```diff
@@ -11,8 +11,8 @@ f.entry(title:_("Port"),field:"port") {
 f.entry(title:_("User name"),field:"userName") {
     f.textbox()
 }
-f.entry(title:_("Password"),field:"password") {
-    f.password(value:instance?.encryptedPassword)
+f.entry(title:_("Password"),field:"secretPassword") {
```

Comment:
Please also have a look at `@QueryParameter("password") String password` in `ProxyConfiguration` [here](https://github.com/jenkinsci/jenkins/pull/3935/files#diff-04a94288d8cb56bee1e9994e4a375d5bR421).